runtime/doc/options.txt

```diff
@@ -6484,7 +6485,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Determines the maximum width to use for the popup menu for completion.
 	When zero, there is no maximum width limit, otherwise the popup menu
 	will never be wider than this value.  Truncated text will be indicated
-	by "..." at the end.  Takes precedence over 'pumwidth'.
+	by an ellipsis "..." at the end.  The ellipsis character(s) can be
+	customized with the 'fillchars' option by setting the "ellipsis:"
+	value. This option takes precedence over 'pumwidth'.
```

Comment:
Better move this to a separate line, otherwise it's not clear if "This option" refers to 'fillchars' or `pummaxwidth`.
```suggestion
	value.
	Takes precedence over 'pumwidth'.
```

Reply:
okay